docs/features/records.md

```diff
@@ -108,35 +174,44 @@ The programmer may provide an explicit constructor with the same signature as th
 At runtime the primary constructor
```

Comment:
Can't leave comment on the paragraph, but line 170: what do non `sealed`, `abstract` record classes have for primary constructors?